README.md

```diff
@@ -23,6 +23,10 @@ Get involved
 * File a [bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Webtools&component=Pontoon&rep_platform=all&op_sys=all)
 * Start hacking on [mentored bugs](https://wiki.mozilla.org/Webdev/GetInvolved/pontoon.mozilla.org)
 
+Deployment
+----------
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
```

Comment:
We should move this under the "Get involved" section. And also replace the "Service documentation" block with another item under the "Get involved" section:
* [Set up Pontoon](http://mozilla-pontoon.readthedocs.io/en/latest/) for local development.